### DISCUSSION TITLE
关于PDF文件显示问题

### DISCUSSION THREAD

**Discussion by chaoskol** [AlistGo/alist#296] [Q&A] [answered]
大大 你好
请问下为何开启PDF显示错误？EPUB目前尚不支援？
要解决以上问题该如何处理，谢谢！
PDF.js v2.12.305 (build: d3e1d7090)
訊息: Load failed

**Comment by xhofe** [comment, marked as answer]
提供一下文件链接，epub暂时不支持

**Reply by chaoskol** [reply]
[文件连结](http://23)

> 提供一下文件链接，epub暂时不支持

**Reply by xhofe** [reply]
请使用https协议，http暂不支持，之后可能会支持。

**Reply by chaoskol** [reply]
好 谢谢！

**Reply by chaoskol** [reply]
另外请教下如何启用Https协议

**Reply by xhofe** [reply]
域名-申请证书-配置文件中编辑证书相关信息或者使用nginx反向代理
